Tests/Source/TestBase.h

```diff
@@ -208,7 +208,17 @@ struct TestBase : public ::testing::Test
 
     void exhaustStackSpace()
     {
-        for (int i = 0; i < 2000000; ++i)
+        // Exhaust LUAI_MAXSTACK = 1000000
+        for (int i = 0; i < 1000000; ++i)
```

Comment:
can we use LUAI_MAXSTACK instead of the hardcoded value ?

Reply:
`LUAI_MAXSTACK` does not exist with Lua 5.1, only `LUAI_MAXCSTACK` does https://github.com/kunitoki/LuaBridge3/blob/dd586f23c39e8bed2a888bc966b2cf3996499c73/Tests/Lua/Lua.5.1.5/src/luaconf.h#L446

Reply:
I see, well it's fine either way